man/paru.conf.5

```diff
@@ -1,55 +1,56 @@
 '\ t
-.TH "PARU.CONF" "5" "2019\-10\-21" "paru v0.0.1" "Paru  Manual"
+.TH "PARU.CONF" "5" "2020\-10\-28" "paru v1.0.0" "Paru Manual"
 .nh
 .ad l
 .SH NAME
-PARU \- AUR Helper
+paru.conf \- paru configuration file
 
 .SH SYNOPSIS
-$PARU_CONF
-.sp
-/etc/paru.conf
-.sp
-~/.config/paru/paru.conf
+/etc/paru.conf, $PARU_CONF, $XDG_CONFIG_HOME/paru/paru.conf, $HOME/.config/paru/paru.conf
```

Comment:
`$PARU_CONF` Has the highest priority, should be first.

That's all I see wrong though.